packages/components/atoms/f-form-field/src/assets/scss/common.scss

```diff
@@ -4,3 +4,6 @@
 $theme: 'jet' !default;
 
 @import  '~@justeat/fozzie/src/scss/fozzie';
+
+$form-input-bg                            : $color-container-default;
+$form-input-bg--hover                     : darken($form-input-bg, $color-hover-01);
```

Comment:
Looks like you need an extra newline under this or the build has a paddy – I think it's a global linting rule

Reply:
🤦 I'll add it in, thanks !